### ISSUE TITLE
Mobs reset/evade range is low

### ISSUE THREAD

**Issue by qwerty4444444444444444444444444** [gamefreedomgit/Frostmourne#201]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (http://wowhead.com/)
[//]: # (wotlk-twinhead.twinstar.cz)

**Description:**
Ever since the server restart on day when legendary items were implemented, Leash range of mobs drastically changed.
If you don't dps them while kitting or pulling other close mob for aoe, they will evade at very low range.

**How to reproduce:**
Pull any mob without dpsing and see when they reset.

**How it should work:**
I don't know exactly what range should be but something has changed definitely.
So basically it should be higher than what it is right now.

**Database links:**

**Comment by Godmatik** [comment]
https://github.com/gamefreedomgit/Frostmourne/issues/145

The leash changes applied on this fix were definitely not done correctly. Sometimes ranged mobs will just reset after several seconds has passed while barely moving at all.
